iox_client_test.go

```diff
@@ -102,6 +103,6 @@ func TestClient(t *testing.T) {
 
 	for reader.Next() {
 		record := reader.Record()
-		t.Logf("%v", record.Column(0).(*array.Uint64).Uint64Values())
```

Comment:
This change is likely due to https://github.com/apache/arrow-datafusion/pull/2636 in upstream datatafusion. 

Thank you for updating the test @domodwyer